fullstack_python/hw-11.py

```diff
@@ -0,0 +1,155 @@
+# task 1
+
+class InfiniteSequence:
+    def __init__(self, sequence):
+        self.sequence = sequence
+        self.index = 0
+
+    def __iter__(self):
+        return self
+
+    def __next__(self):
+        result = self.sequence[self.index]
+        self.index = (self.index + 1) % len(self.sequence)
+        return result

```

Comment:
You should use a generator function

In addition, the sequence should be infinite, in your case it is limited by count

In fact, it should only accept numbers that will be output